src/pages/HomeView/index.js

```diff
@@ -18,6 +21,10 @@ const useStyles = makeStyles((theme) => ({
 export default function HomeView(props) {
   const classes = useStyles();
 
+  useEffect(() => {
+    SplashScreen.hide();
+  }, []);
+
```

Comment:
I think @rizwan2000rm  you can avoid using `useEffect`  to hide the splash screen where you can do this by making  `launchAutoHide: true` in the `capacitor.config.json` file right!